docs/Tutorials/Privacy/Configuring-Multi-Tenancy.md

```diff
@@ -0,0 +1,112 @@
+tls = "off"
+```
+
+## 5. Start Orion Nodes
+
+In each `Orion` directory, [start Orion](Configuring-Privacy.md#5-start-orion-nodes) and specify the configuration file.
+
+## 6. Start Besu Node-1
+
+In the `Node-1` directory, start Besu Node-1:
+
+```bash tab="MacOS"
+besu --data-path=data --genesis-file=../genesis.json --rpc-http-authentication-enabled --rpc-http-authentication-jwt-public-key-file=publicKey.pem --rpc-http-enabled --rpc-http-api=ETH,NET,IBFT,EEA,PRIV --host-whitelist="*" --rpc-http-cors-origins="all" --privacy-enabled --privacy-url=http://127.0.0.1:8888 --privacy-multi-tenancy-enabled --min-gas-price=0   
+```
+
+The command line specifies privacy options: 
+
+* [`--rpc-http-authentication-enabled`](../../Reference/CLI/CLI-Syntax.md#rpc-http-authentication-enabled) enables authentication
+for JSON-RPC APIs.
+* [`--rpc-http-authentication-jwt-public-key-file`](../../Reference/CLI/CLI-Syntax.md#rpc-http-authentication-jwt-public-key-file) specifies the Operator's [public key file](#1-generate-a-private-and-public-key-pair). Used to authenticate the [tenant JWT tokens](#6-generate-tenant-jwt-tokens).
+* [`--privacy-enabled`](../../Reference/CLI/CLI-Syntax.md#privacy-enabled) enables privacy.
+* [`--privacy-url`](../../Reference/CLI/CLI-Syntax.md#privacy-url) specifies the Orion node URL (`clienturl` in `orion.conf`)
+* [`--privacy-multi-tenancy-enabled`](../../Reference/CLI/CLI-Syntax.md#privacy-multi-tenancy-enabled) enables multi-tenancy.
+
+!!! note
+    [`--rpc-http-authentication-jwt-public-key-file`](../../Reference/CLI/CLI-Syntax.md#rpc-http-authentication-jwt-public-key-file) is only
+    required when using [JWT public key authentication](../../HowTo/Interact/APIs/Authentication.md#jwt-public-key-authentication). If
+    using [username and password authentication](../../HowTo/Interact/APIs/Authentication.md#username-and-password-authentication),
+    use [`--rpc-http-authentication-credentials-file`](../../Reference/CLI/CLI-Syntax.md#rpc-http-authentication-credentials-file) instead.
+
+You may now [start the remaining Besu nodes](Configuring-Privacy.md#7-start-besu-node-2).
+
+## 7. Generate Tenant JWT Tokens
+
+[Generate the JWT token](../../HowTo/Interact/APIs/Authentication.md#2-create-the-jwt-token) for each tenant and specify the [tenant's Orion public key](#2-generate-orion-keys)
+in the `privacyPublicKey` field.
+
+Ensure the appropriate [JSON-RPC API permissions](../../HowTo/Interact/APIs/Authentication.md#json-rpc-permissions) are applied to the token. For
+example, ensure the `PRIV` and `EEA` APIs are enabled for privacy.
+
+!!! note
+    This step is not required when using [username and password authentication](../../HowTo/Interact/APIs/Authentication.md#username-and-password-authentication)
+    to create the required JWT tokens.
+    
+  You can now [use the authentication token to make requests](../../HowTo/Interact/APIs/Authentication.md#using-an-authentication-token-to-make-requests).
```

Comment:
You can now use ... -> Use ... 

Suggested change to make more direct/simplify language

Reply:
Done